ckanext/qgov/common/views/user.py

```diff
@@ -39,8 +40,16 @@ def user_edit_override():
     return EditView().dispatch_request()
 
 
+def _gettext_wrapper(key):
+    translation = original_gettext(key)
+    if key == 'Login failed. Bad username or password.':
+        translation = translation.replace('or password.', 'or password or reCAPTCHA.')
```

Comment:
Unsure if we still need this 'logged' in override.
https://github.com/qld-gov-au/ckanext-qgov/commit/1185b4c87b8bddf5ca49924d3169b38224b77573#diff-efa22cf47b1e8aa51b8cb0c79a87d1597c06fa9f71611bd8b95b07a8e3668910R24

Reply:
Huh. I thought I'd tested it locally on CKAN 2.9 and it didn't work without keeping that intercept, but I must have been doing something wrong that time, because yes, it works. Okay, taking that out.